results/server/result_accumulator.py

```diff
@@ -1,36 +1,41 @@
 import pytest
-import git
 from jinja2 import Template
 from tag import Tag
+from repo_handler import RepoHandler
 from typing import Union
 import yaml
 import jsonschema
 import json
 import datetime
 import os
 
+# TODO: move this to a config file
+pages_repo_dir="../macfe-hil.github.io"
+pages_branch="main"
+git_username="macformularacing"
+git_email="macformulaelectric@gmail.com"
+
 class ResultAccumulator:
     def __init__(self, 
                  tags_fp: str, 
                  tags_schema_fp: str, 
                  template_fp: str,
                  historic_tests_fp: str, 
                  reports_dir: str,
-                 pages_repo_dir: str,
-                 pages_branch: str,
-                 git_username: str,
-                 git_email: str) -> None:
+                 ) -> None:
         # Generate tag database from yaml file
         self.__parse_args(tags_fp, tags_schema_fp)
         self.tag_submissions: dict[str, any] = {}  # tag_id -> value cache
         self.error_submissions: list[str] = []  # list of cached errors
         self.historic_tests_fp = historic_tests_fp
         self.template_fp = template_fp
         self.reports_dir = reports_dir
-        self.pages_repo_dir = pages_repo_dir
-        self.pages_branch = pages_branch
-        self.git_username = git_username
-        self.git_email = git_email
+        self.repo_handler = RepoHandler(
```

Comment:
Do this at the top level and pass it into the result accumulator (hence dependency injection)